control/lti.py

```diff
@@ -157,48 +157,31 @@ def timebase(sys, strict=True):
 
     return sys.dt
 
-# Check to see if two timebases are equal
-def timebaseEqual(sys1, sys2):
```

Comment:
Removing this function here and from `__all__` may break existing code. Maybe introduce a `DeprecationWarning` between 0.9 and 0.10 or 0.9.1?

Reply:
Done.